conda/recipes/dask-cuda/meta.yaml

```diff
@@ -2,12 +2,12 @@
 
 # Usage:
 #   conda build -c conda-forge -c defaults .
-{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
```

Comment:
I think we still need this as OPS uses it to create nightly packages

Reply:
Do you know if the `version` set in `setup.py`

https://github.com/rapidsai/dask-cuda/blob/3c6e0a9d18615376c05074622b0cec2fc1189aeb/setup.py#L12-L16

Matches with the `version` set in this line? Then we could just do something like

```
{% set version = data.get('version', '0.0.0.dev') %}
```

Reply:
I'm not sure that they match. In `setup.py` it will be prefixed with `v` if I'm not mistaken, but conda certainly disallows that.

I personally don't see a need to change the conda recipe since it matches the rest of RAPIDS.

Reply:
The main idea here was to reduce duplication between `setup.py` and the Conda recipe, which is more a convenience than anything else.

If you think this is an unnecessary change, I'm happy to keep the original Conda recipe, although I do think that the [`requirements.txt`](https://github.com/rapidsai/dask-cuda/blob/branch-0.20/requirements.txt) in this repo should be updated to align with it.

Reply:
Maybe let's stick to the `requirements.txt` change for now